app/models/registration_wizard.rb

```diff
@@ -206,11 +206,11 @@ def answers
     end
     unless eligible_for_funding?
       if course.ehco?
-        array << OpenStruct.new(key: "How is your EHCO being paid for?",
+        array << OpenStruct.new(key: "How are you funding your course?",
```

Comment:
@dalcJ I have added this file as well, although the copy should change, I think.